src/Core/Vipr.Core/CodeModel/OdcmDeprecation.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Vipr.Core.CodeModel
+{
+    public class OdcmDeprecation
+    {
+        /// <summary>
+        /// String for Deprecation Description
+        /// </summary>
+        public string Description { get; set; }
+
+        /// <summary>
+        /// String for Deprecation Version
+        /// </summary>
+        public string Version { get; set; }
```

Comment:
RemovalDate would be helpful to customers to add to /// comments so that it gets propagated to intellisense - it would help them plan to make their change. I'd like this information to be added to the obsolete message in the templates

Reply:
See above. I would prefer to have standard guidance that workloads include this in the deprecation message, rather than try to artificially augment the messages from the workloads.